docs/cobbler.rst

```diff
@@ -930,10 +930,38 @@ Example:
 
     $ cobbler aclsetup
 
-Cobbler buildiso
+cobbler buildiso
```

Comment:
```suggestion
Cobbler buildiso
```
Cobbler is always written in capital letters throughout the documentation.

Reply:
I disagree. It is always written in lower case letters when referring to a command and this is the case in this heading.

Reply:
You are right, but I thought this is a headline and therefore we write it in capital letters like e.g. [here](https://cobbler.readthedocs.io/en/latest/cobbler.html#cobbler-buildiso).

Reply:
Okay if we do that we need to change more places. I will have a look